src/core/web_api/p2_api.js

```diff
@@ -100,20 +105,16 @@ class Auth0APIClient {
   }
 
   signUp(options, cb) {
-    const { popup, sso } = this.authOpt;
-    const { autoLogin } = options;
-
     delete options.autoLogin;
-
-    this.client.signup(options, (err, result) => cb(err, result));
+    this.client.signup(trimAuthParams(options), (err, result) => cb(err, result));
   }
 
   resetPassword(options, cb) {
-    this.client.changePassword(options, cb);
+    this.client.changePassword(trimAuthParams(options), cb);
```

Comment:
Similar feedback here ... there might be another transform in the future, in which case this starts to get really hard to read.